style.css

```diff
@@ -0,0 +1,731 @@
+* {
+    margin: 0;
+}
+
+.container {
+    /* height: 100vh; */
+    display: grid;
+    grid-template:
+        'header main news' /
+        25% 1fr 30%
```

Comment:
It can be difficult to work with %'s for layouts, it's better to define fixed pixel widths for side bars and change them with media queries at certain break points

Reply:
Oh god! And I thought that %'s will be better for responsive design since analogy stays constant no matter what the actual size is. Duly noted as are the rest of your remarks.